scripts/vci-directory-auditor/src/assemble.ts

```diff
@@ -0,0 +1,91 @@
+// Script to assemble the VCI issuers directory snapshot
+
+import { Command } from 'commander';
+import fs from 'fs';
+import { DirectoryLog, IssuerLogInfo } from './interfaces';
+
+interface Options {
+    snapshot: string;
+    current: string;
+    new: boolean;
+}
+
+//
+// program options
+//
+const program = new Command();
+program.requiredOption('-s, --snapshot <snapshot>', 'path to the previous directory snapshot to update');
+program.requiredOption('-c, --current <current>', 'path to current snapshot to integrate in the previous snapshot');
+program.option('-n, --new', 'create a new directory snapshot from the current snapshot');
+program.parse(process.argv);
+const options = program.opts() as Options;
+
+function updateDirectorySnapshot(snapshot: DirectoryLog, current: DirectoryLog) : DirectoryLog {
+    if (snapshot.directory !== current.directory) {
+        // uncommon scenario, so let's warn about that
+        console.log("Warning current log is not from the same directory");
+    }
+
+    if (snapshot.time > current.time) { // TODO: will that work?
```

Comment:
Would this likely indicate a developer error? If so, might be helpful to fail hard.

Reply:
I was imagining someone might want to build a snapshot from multiple sources, but yeah, in the absence of another public directory, I think it might indeed be better to fail here. I'll update the PR.